laws/shared/src/test/scala/algebra/laws/LawTests.scala

```diff
@@ -191,5 +191,6 @@ trait LawTestsBase extends FunSuite with Discipline {
   }
 
   laws[OrderLaws, Int]("fromOrdering").check(_.order(Order.fromOrdering[Int]))
+  laws[OrderLaws, Array[Int]].check(_.order)
```

Comment:
Me neither actually. I guess just make a test and use scala check like normal. Something like:

```scala
forAll { (a: Array[Int], b: Array[Int]) =>
  Order[Array[Int]].compare(a, b) == Ordering.Iterable[Int](a.toList, b.toList)
}
```
should do it.